src/settings/index.js

```diff
@@ -102,6 +103,12 @@ const pages = [
     component: PluginSurface,
     perm: 'ui-developer.settings.plugin-surface',
   },
+  {
+    route: 'handler-surface',
+    labelId: 'ui-developer.handler-surface',
+    component: HandlerSurface,
+    // perm: 'ui-developer.settings.handler-surface',
```

Comment:
Please enable this before you merge.

Reply:
If I enable the permission, I won't be able to use the feature.